_layouts/default.html

```diff
@@ -254,8 +254,26 @@ <h2>Related articles</h2>
       anchors.add().remove('.subfooter h1, .subfooter h2');
     </script>
   {% endif %}
+  <script src="{{ '/assets/js/timeme.min.js' | relative_url }}"></script>
+  <script type="text/javascript">
+    // Initialize library and start tracking time
```

Comment:
@RasonJ why do we have this initializing twice?   Also, I don't think we need the comments...  Normally I love comments, but what it does appears pretty clear...